.github/actions/setup-tauri/action.yml

```diff
@@ -1,6 +1,11 @@
 # From https://tauri.app/v1/guides/getting-started/prerequisites
 name: "Setup Tauri"
 description: "Sets up the dependencies for building and testing Tauri apps"
+inputs:
+  runtime:
+    description: "Whether to also set up runtime dependencies"
+    required: false
+    default: false
```

Comment:
Yeah Actions yaml can be weird about booleans. Might be better to make this an explicit string?

Reply:
Hmm that feels wrong :sweat_smile: 

Let me see if what I just pushed works now.